### PR TITLE
Fix greeting

### LINKED ISSUE #604: Fix GitHub icon and greeting not showing properly
## Issue
- GitHub icon height and width has some issues
- If the user is logged in then greeting is not shown
image

### PR DESCRIPTION
**Pull request by shubhdevelop**
### Issue

Closes #604

### What is the changed
- GitHub image now have a width and height 
- on debug there is greeting in the navbar is user is logged in

### Is Development Tested?

- [ ] Yes
- [x] No

### Before / After Change Screenshots
**Before :**
image

**After:**
image

### CODE REVIEW

File: debug.html
Comment (rohan09-raj) [inline debug.html:1]: @shubhdevelop Can we please add the shimmer for `Sign in with GitHub` like home page ??
You can check it by visiting this https://realdevsquad.com/

File: debug.html [hunk 1]
```diff
@@ -268,7 +273,10 @@ <h4>Apply privileges :</h4>
         >
       </p>
     </footer>
+    <script src="/js/utils.js"></script>
```
Comment (rohan09-raj) [inline debug.html:276]: Why this is being added ?

Comment (shubhdevelop) [reply debug.html:276]: These were the necessary files needed for  greeting  to function properly

File: debug.html
Comment (shubhdevelop) [reply debug.html:1]: Sure

**Review by rohan09-raj — APPROVED** [review]

**Pull request merged**